opal/mca/pmix/pmix3x/pmix/src/hwloc/hwloc.c

```diff
@@ -48,7 +48,7 @@ static bool external_topology = false;
 
 #if HWLOC_API_VERSION >= 0x20000
 static size_t shmemsize = 0;
-static size_t shmemaddr;
+static uintptr_t shmemaddr;
```

Comment:
It looks like this value is serialized as a `size_t` down below.  Can you investigate?  I don't know where the corresponding deserialization occurs -- you might need to chase that down, too.

I didn't check further to see how the other variables that were converted to `uintptr_t` are used; those might need to be updated in their usage, too.

Reply:
@nmorey Any update on this?